.circleci/config.yml

```diff
@@ -26,7 +26,12 @@ aliases:
   install_node_modules: &install_node_modules
     run:
       name: Install node modules
-      command: yarn --frozen-lockfile
+      command: yarn
```

Comment:
you still want to install with frozen lockfile
```suggestion
      command: yarn --frozen-lockfile
```

Reply:
Actually no, because there seem to be a bug with `yarn` that it keeps `yarn.lock` locked but doesn't exit with non-zero code when it should be updated

Reply:
So this is exact reason why we were not getting errors about dirty lock file

Reply:
Ah, my guess was that it would still create a dirty lock file. My bad.